examples/advanced/experiment-tracking/mlflow/hello-pt-mlflow-client/README.md

```diff
@@ -0,0 +1,193 @@
+
+---
+
+## How It Works
+
+This example demonstrates **client-side tracking** where each client keeps its own metrics locally.
+
+### Step 1: Logging Metrics (in `client.py`)
+
+Your training script logs metrics using NVFlare's tracking API:
+
+```python
+from nvflare.client.tracking import MLflowWriter
+
+mlflow = MLflowWriter()
+mlflow.log_metric("accuracy", acc, step=epoch)
+```
+
+This creates a **local event** (`analytix_log_stats`) on the **NVFlare Client** side.
+
+### Step 2: Local Event Handling
+
+Unlike server-side tracking, there is **no event conversion** to federated events:
+
+1. **`MLflowReceiver`** (deployed to each NVFlare Client)
+   - Listens for **local event**: `analytix_log_stats` (NOT `fed.analytix_log_stats`)
+   - Writes metrics to **local MLflow directory** on that client
+   - Metrics **never leave the client**
+
+2. **No `ConvertToFedEvent` widget**
+   - Local events stay local
+   - Server never receives metrics
+
+### Result
+
+Each client has its own **separate MLflow instance** with only its own metrics. Complete data privacy!
+
+### Key Terminology
+
+To avoid confusion:
+- **`client.py`**: Your training script (user code that logs metrics)
+- **NVFlare Client**: The FL client runtime that executes your training script
+- **Client-Side Tracking**: Receiver deployed on NVFlare Client (this example)
+- **Local Event**: `analytix_log_stats` - stays on the client
+- **Federated Event**: `fed.analytix_log_stats` - sent to server (NOT used in this example)
+
+---
+
+## Comparison: Server vs Client Tracking
+
+| Aspect | Server-Side | Client-Side (This Example) |
+|--------|-------------|----------------------------|
+| **Metrics Location** | Centralized on server | Distributed per site |
+| **Event Type** | `fed.analytix_log_stats` | `analytix_log_stats` |
+| **Receiver Location** | Server | Each client |
+| **MLflow UI** | Single instance | One per site |
+| **Privacy** | Server sees all metrics | Sites keep metrics local |
+| **Use Case** | Centralized monitoring | Site-specific analysis |
+
+---
+
+## Customization
+
+### Change Tracking Location
+
+```python
+tracking_uri = f"file:///my/custom/path/{site_name}/mlruns"
+```
+
+### Add Experiment Tags
+
+```python
+receiver = MLflowReceiver(
+    tracking_uri=tracking_uri,
+    events=[ANALYTIC_EVENT_TYPE],
+    kw_args={
+        "experiment_name": f"{site_name}-experiment",
+        "run_name": f"{site_name}-run-001",
+        "experiment_tags": {"site": site_name, "privacy": "local"},
+    }
+)
+```
+
+### Remote MLflow Server
+
+```python
+tracking_uri = "http://mlflow-server.site-1.local:5000"
+```
+
+---
+
+## Additional Resources
+
+- [MLflow Documentation](https://www.mlflow.org/docs/latest/index.html)
+- [Experiment Tracking Guide](https://nvflare.readthedocs.io/en/main/programming_guide/experiment_tracking.html)
+- [Recipe API Documentation](https://nvflare.readthedocs.io/en/main/programming_guide/job_recipes.html)
```

Comment:
we dont have this link,
need to change to https://nvflare.readthedocs.io/en/main/user_guide/data_scientist_guide/job_recipe.html